### ISSUE TITLE
Not Working on YouTube

### ISSUE THREAD

**Issue by RafaelDominiquini** [CanisLupus/swift-selection-search#185]
**Describe the problem**
The popup is not showing in any YouTube page, in any text...

**How to reproduce the problem**
1. Go to 'www.youtube.com'
2. Selected any text
3. Click 'Ctrl+Shift+Space'
4. Nothing happens!

**Anything else you'd like to include**
In any other site appear to be working ok!

**System info:**
- Browser: Firefox
- Browser version: 74.0
- System: Windows 10

**Comment by CanisLupus** [comment]
Hi @RafaelDominiquini, thank you very much for the report. This was caused by a change I made in the positioning of the popup in the previous update (to fix something else). YouTube apparently has a body height of 0 in the main area and I had not considered this.

I've fixed it and submitted a new version (3.41.2) to the addons website. We'll need to wait for the review and then it should be set!

Thanks again!
Daniel
